xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -383,7 +383,7 @@ The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original wri
 
 ## Remarks
 
-The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original writer options but now writes to `bufferWriter` as the new destination.
+The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original writer options but now write to `bufferWriter` as the new destination.
```

Comment:
Why did you remove the singular "s"? Utf8JsonWriter is still a singular.

Reply:
Seemed to make more grammatical sense to my ear.

Reply:
I'm on the fence on this one. @rpetrusha can you help?

Reply:
I agree with @Jozkee, The Utf8JsonWriter is who's writing. It writes.

Reply:
The other option would be to add will write

Reply:
The tense of write is future, so it should be "write" rather than "writes". As Maira points out, it could also be changed to "will write", and that's fine, but it's not necessary.